### DISCUSSION TITLE
Multiple warnings on logs

### DISCUSSION THREAD

**Discussion by KuasarCloud** [apache/cloudstack#9526] [Q&A] [answered]
Hi. Community,

Do you know how to manage these warnings in logs

**Agent**
- WARN [kvm.resource.LibvirtComputingResource] (agentRequest-Handler-2:null) (logid:3e3293b2) Couldn't retrieve free memory, returning -1.

**Management**
- Failure during validating cluster request from 192.168.x.x
- unable to find vm disk stats from host for account: 1 with vmId: 699 and volumeId:2255

**Comment by DaanHoogland** [comment, marked as answer]
@KuasarCloud , it is a warning so if you don't have any hampered functionality, ignore it.
The issues you mention under **Management**, are these from the logs, as well?

To trouble shoot, you can go to the host and do a `virsh dumpxml ##` for the vm. (you can find the right ## by doing `visrh list | grep <vmname>`
